### PR TITLE
MacOS tweaks update to Sonoma

### PR DESCRIPTION
**Pull request by julesvanrie**
The macOS tweaks at the end are written for an older version of macOS.

This PR updates the instructions to work out of the box in Sonoma 14.5.

Updated:
- EN
- ES (was easy because it is written for EN macOS)

Not updated because I don't want to install FR/PT languages on my mac to never use it:
- FR
- PT
Maybe someone else can check?

### CODE REVIEW

**Comment by julesvanrie** [comment]
I was looking into this because I'd like to include this part in the data-setup. The partial is currently not included.

**Comment by ajdubovoy** [comment]
Sorry for being slow on this! Will look next week

File: _partials/es/macos_settings.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
-También puedes ir a ` > System Preferences > Mission Control` y haz clic en el botón `Hot Corners` abajo a la izquierda. Selecciona la opción donde al hacer clic en la esquina inferior derecha se active el protector de pantalla. De esa manera al dejar tu escritorio podrás bloquear tu pantalla rápidamente poniendo el cursor del ratón en la esquina inferior derecha. En 5 segundos tu Macbook se bloqueará y te pedirá la contraseña para volver a tu sesión.
+También puedes ir a ` > System Settings... > Desktop & Dock` y haz clic en el botón `Hot Corners...` abajo a la izquierda. Selecciona la opción donde al hacer clic en la esquina inferior derecha se active el protector de pantalla. De esa manera al dejar tu escritorio podrás bloquear tu pantalla rápidamente poniendo el cursor del ratón en la esquina inferior derecha. En 5 segundos tu Macbook se bloqueará y te pedirá la contraseña para volver a tu sesión.
```
Comment (ajdubovoy) [inline _partials/es/macos_settings.md:7]:
```suggestion
También puedes ir a ` > System Settings... > Desktop & Dock` y hacer clic en el botón `Hot Corners...` abajo a la izquierda. Selecciona la opción donde al hacer clic en la esquina inferior derecha se active el protector de pantalla. De esa manera al dejar tu escritorio podrás bloquear tu pantalla rápidamente poniendo el cursor del ratón en la esquina inferior derecha. En 5 segundos tu Macbook se bloqueará y te pedirá la contraseña para volver a tu sesión.
```

**Review by ajdubovoy — APPROVED** [review]
Ok this is great, thank you! I noticed a small grammar error in the Spanish

**Comment by ajdubovoy** [comment]
Checking other langs now

**Comment by ajdubovoy** [comment]
OK I think I got the French and Portuguese right by looking through the Apple docs for each section in those languages (it was a process lol)

**Comment by ajdubovoy** [comment]
@ssaunier does this seem OK to merge?

**Review by ssaunier — APPROVED** [review]
🙏

**Pull request merged**